### PR TITLE
Refine budget limits per account

### PR DESCRIPTION
**Pull request by BrunoGrandePhD**
We've increased the amount of activity in nextflow-prod, so the budget limits needed to be tweaked. I've also noticed that GuardDuty and AWS Config costs scale roughly with compute, so I categorized them separately to keep better track of other costs. In this PR, I move the budget limits to the config files and tailor them per account.

### CODE REVIEW

File: config/infra-dev/config.yaml [hunk 1]
```diff
@@ -1,2 +1,5 @@
 sso_admin_role:
   arn: arn:aws:iam::035458030717:role/aws-reserved/sso.amazonaws.com/AWSReservedSSO_Administrator_580e9f32ac55c4e7
+budgets:
+  compute: 1000
```
Comment (thomasyu888) [inline config/infra-dev/config.yaml:4]: Just as a reminder for myself, these budgets override what's in `config/config.yaml` correct?

**Review by thomasyu888 — APPROVED** [review]
LGTM

File: config/infra-dev/config.yaml (hunk 1 above)
Comment (BrunoGrandePhD) [reply config/infra-dev/config.yaml:4]: Yup! https://docs.sceptre-project.org/dev/docs/stack_group_config.html#stack-group-config-cascading-config

File: config/infra-ampad/config.yaml [hunk 2]
```diff
@@ -2,3 +2,6 @@
 # it sits outside of our AWS Organization fo STRIDES billing
 sso_admin_role:
   arn: arn:aws:iam::751556145034:role/strides-ampad-workflows-admin
+budgets:
+  compute: 5000
+  storage: 2000
```
Comment (zaro0508) [inline config/infra-ampad/config.yaml:7]: the same values are in `config/config.yaml` so do we need this here?

File: src/sceptre/variables/ampad.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+compute_budget: 5000
+storage_budget: 2000
+security_budget: 500
+other_budget: 500
```
Comment (zaro0508) [inline src/sceptre/variables/ampad.yaml:4]: make strings instead of numbers

**Review by zaro0508 — CHANGES_REQUESTED** [review]

**Review by zaro0508 — APPROVED** [review]

**Pull request merged**